projects/EWTeam/scripts/ewrun.bat

```diff
@@ -4,6 +4,12 @@ REM Copyright (C) 1989-2019 PC2 Development Team: John Clevenger, Douglas Lane,
 rem Purpose: Start the ew Team server/java bridge 
 rem Author : pc2@ecs.csus.edu
 
+echo.
+echo Warning: EWTeam is deprecated and will be removed in an upcoming version of PC^2.  
```

Comment:
You should use PC^^2 if you want to see the caret in echo commands.  Caret is the escape char for the windows command interpreter.  If you "quote" the string, you'll see the quotes... So, ^^ works.

Reply:
> You should use PC^^2 if you want to see the caret in echo commands. Caret is the escape char for the windows command interpreter. If you "quote" the string, you'll see the quotes... So, ^^ works.

Done.  (In case anyone wonders, I DID test the batch file before submitting the PR.  However, I can't swear that it didn't say "PC2" instead of "PC^2"; I'm so used to seeing both forms that I could easily have overlooked that detail.)  I've submitted an update to the PR.